app/helpers/application_helper.rb

```diff
@@ -3,6 +3,18 @@ def page_text_direction
     I18n.t("i18n.direction", locale: I18n.locale, default: "ltr")
   end
 
+  def t_locale_fallback(key, options = {})
+    options['locale'] ||= I18n.locale
```

Comment:
Are we ever setting `options['locale']` outside of the tests? It would be great not to have logic specific to tests. If you want to set the locale inside your test, you should be able to set it: `I18n.locale = :de` (for example).

Reply:
We don't currently, no, but when I set I18n.locale = :de it caused a cascade of failing tests because I couldn't figure out how to unset it afterwards.

Reply:
Are you able to do something like this:

```
I18n.with_locale(:de) do
    assert .....
end
```

Reply:
Your test suggestion is working, thanks!